src/ui/view/modal/okp4-modal.scss

```diff
@@ -0,0 +1,37 @@
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-okp4-modal-main {
+  @include with-theme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: themed('text');
+    padding: 12px;
+    width: 358px;
+
+    @include use-breakpoints(('mobile', 'tablet')) {
+      width: 352px;
+    }
+
+    .okp4-dataverse-portal-okp4-modal-top-element {
+      max-height: 78px;
+    }
+
+    .okp4-dataverse-portal-okp4-modal-top-element-divider {
+      border-top: 1px solid themed('modal-line-divider');
+      width: 358px;
```

Comment:
divider must take 100% modal's width according to the mockups, see here:

![Capture d’écran 2023-08-11 à 14 08 54](https://github.com/okp4/dataverse-portal/assets/35332974/bfc20fd8-015b-4713-9beb-94cf9b9d61e4)

what we have: 

![Capture d’écran 2023-08-11 à 14 09 31](https://github.com/okp4/dataverse-portal/assets/35332974/df663527-491e-4936-9f78-2eae8daf643e)

Reply:
Ah the padding was the problem, fixed now!